common/lib/client/auth.js

```diff
@@ -437,11 +437,6 @@ var Auth = (function() {
 			tokenRequest(tokenRequestOrDetails, function(err, tokenResponse, headers, unpacked) {
 				if(err) {
 					Logger.logAction(Logger.LOG_ERROR, 'Auth.requestToken()', 'token request API call returned error; err = ' + Utils.inspectError(err));
-					if(!err.code) {
-						/* network errors don't have an error code, so assign them
-						 * 40170 so they'll be seen by connectionManager as nonfatal */
-						err = new ErrorInfo(Utils.inspectError(err), 40170, 401);
-					}
```

Comment:
Note: removed this as errors.json technically reserves 40170 for errors from the client token callback, which this isn't. (Having only added it recently to stop network errors getting a token being fatal, which is no longer needed)